client/views/index.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <div>
+        <form action="http://localhost:4000/register">

```

Comment:
I know that this is a toy client, but this should not be hardcoded here. From the app point of view, you only need the path not the absolute url.